src/main/scala/edu/twitter/SentimentAnalyzer.scala

```diff
@@ -32,15 +33,27 @@ object SentimentAnalyzer extends App {
   val modelService = new ModelService(new GradientBoostingBuilder(sc))
   modelService.start()
 
+  val indexHandler = new IndexHandler
+  val result = indexHandler.create("twitter", "sentiment")
+
+  var indexName = ""
+
+  result match {
+    case Left(failure) => terminate()
+    case Right(name) => indexName = name
+  }
+
   val classifier = new Classifier(ssc)
   val classifiedStream = classifier.createClassifiedStream(GradientBoostingModel.name)
-  classifiedStream.foreachRDD(EsSpark.saveToEs(_, "twitter/sentiment"))
+  classifiedStream.foreachRDD(EsSpark.saveToEs(_, indexName))
```

Comment:
I believe we can't now create indices with dynamic names. so I would recommend adding a field in the `ClassifiedTweet` pointing to the `model` used for classification.

Reply:
can't understand why we can't create indices with dynamic names?
we can have two indices for each model.

Note:
index name and mapping name must be lowercase characters.

Reply:
This can be a way to go, but you will need a schema for each index in `resources` folder right?, another way (could be easier) is to add a string in the `ClassifiedTweet` pointing to the model used for classification.